src/common/models/ProjectionsPair.ts

```diff
@@ -119,7 +119,7 @@ export function getDataset(projection: Projection, metric: Metric): Column[] {
     case Metric.HOSPITAL_USAGE:
       return projection.getDataset('icuUtilization');
     case Metric.VACCINATIONS:
-      return projection.getDataset('vaccinationsBivalentBoostedFall2022');
+      return projection.getDataset('vaccinationsCompleted');
```

Comment:
Drive-by change to allow us to display charts when we don't have booster data, e.g.:

before:
<img width="963" alt="image" src="https://github.com/act-now-coalition/covid-act-now-website/assets/55333380/c4824f60-9a64-458d-8242-58fc2c67c4ff">

after:
<img width="963" alt="image" src="https://github.com/act-now-coalition/covid-act-now-website/assets/55333380/98b45ffa-6ee5-457b-90f5-9d2be7a220d9">